MISRA.md

```diff
@@ -0,0 +1,26 @@
+# MISRA Compliance
+
+The coreMQTT Agent library files conform to the [MISRA C:2012](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx)
+guidelines, with some noted exceptions. Compliance is checked with Coverity static analysis.
+Deviations from the MISRA standard are listed below:
+
+### Ignored by [Coverity Configuration](tools/coverity/misra.config)
+| Deviation | Category | Justification |
+| :-: | :-: | :-: |
+| Directive 4.5 | Advisory | Allow names that MISRA considers ambiguous (such as LogInfo and LogError) |
+| Directive 4.8 | Advisory | Allow inclusion of unused types. Header files from coreMQTT, which are used by the agent files, may define types that are not used by the agent. |
```

Comment:
It seems like the portion inside the commas just repeats information that's already there.
```suggestion
| Directive 4.8 | Advisory | Allow inclusion of unused types. Header files from coreMQTT may define types that are not used by the agent. |
```

Reply:
Sure